signer/src/config/default.toml

```diff
@@ -148,6 +148,11 @@ bootstrap_signing_set = [
 # 
 # Required: true Environment: SIGNER_SIGNER__BOOTSTRAP_SIGNATURES_REQUIRED
 bootstrap_signatures_required = 2
+
+# Seconds to wait before processing a new Bitcoin block.
+# Required: true Environment: SIGNER_SIGNER__BITCOIN_PROCESSING_DELAY
+bitcoin_processing_delay = 0
```

Comment:
Setting this to 0, but we should have a non-zero default when shipping to prod.

Reply:
Could you just create a new ticket for this in the milestone as a "don't forget" and link it in a `# TODO: xxx` here?

Reply:
Done in https://github.com/stacks-network/sbtc/issues/715.